components/list-item-meeting.json

```diff
@@ -0,0 +1,21 @@
+{
+  "listitemMeeting": {
```

Comment:
when I read this name, I thought it was a list item inside a meeting/call (maybe a participant's item). Then I understood it was the listItem that represents a scheduled meeting (normally in the Meetings tab). Could we change this name to be either:
- `sheduledmeetingItem`
- `meetingItem`
?
(I took out the 'list' part of the name because it could cause confusion with the component in `components/list-item`

Reply:
Hi @nataliadelmar, yes it represents a meeting row/bricklet in the Meetings List table in the meetings tab, when you select "List" view.
I followed the pattern for "list-item", hence having "list" in the title, but can remove that

Reply:
thank you!